### PR TITLE
fix(VPN): Trafic description and configuration possibilities

### PR DESCRIPTION
**Pull request by 4rthurLegrand**

### CODE REVIEW

File: modules/ROOT/pages/VPNConfiguration.adoc [hunk 1]
```diff
@@ -33,7 +34,7 @@ To request a VPN connection with Bonita Cloud, create a new case on the Customer
 * VPN gateway type
 * Gateway public IP address
 * Encryption algorithms (AES-128, AES-256)
-* Integrity Algorithms (SHA1, SHA2-256)
-* Authentication modes (pre-shared key/digital signature)
-* Diffie-Hellman group (2, 5, 14...)
+* Integrity Algorithms (SHA2-256, SHA2-384, SHA2-512)
+* Authentication modes (IKEv1, IKEv2 - used for pre-shared key/digital signature)
```
Comment (JeremJR) [inline modules/ROOT/pages/VPNConfiguration.adoc:38]: Digital signature isn't for VPN using certificates?
```suggestion
* Authentication modes (IKEv1, IKEv2 - used for pre-shared key)
```

File: modules/ROOT/pages/VPNConfiguration.adoc [hunk 2]
```diff
@@ -33,7 +34,7 @@ To request a VPN connection with Bonita Cloud, create a new case on the Customer
...
+* Diffie-Hellman group (2, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24)
```
Comment (JeremJR) [inline modules/ROOT/pages/VPNConfiguration.adoc:39]: As you removed 5, we may also remove 2
```suggestion
* Diffie-Hellman group (14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24)
```

File: modules/ROOT/pages/VPNConfiguration.adoc [hunk 3]
```diff
@@ -12,13 +12,14 @@ Bonita Cloud supports Internet Protocol security (IPsec) VPN connections.
...
 
-To ensure a versatile connectivity from anywhere, Bonita Cloud will continue to allow connection from public internet. This is done through a strong secured http link (supporting TLS 1.3).
+To ensure a versatile connectivity from anywhere, Bonita Cloud will continue to allow connections from public internet. This is done through a secured HTTPS link (supporting TLS 1.3).
 
-== Outgoing Traffic
+== Outgoing Traffic (from Bonita Cloud POV)
 
-Bonita Cloud will still allow connections towards the public internet but will also be capable to route securely connection towards your private network.
+Bonita Cloud will be able to securely connect to your internal resources which are not exposed on the Internet.
+Bonita Cloud VPN is designed a way that only process connectors can reach your private resources. This means that Bonita Cloud users' trafic will not be embeded inside VPN connection.
```
Comment (JeremJR) [inline modules/ROOT/pages/VPNConfiguration.adoc:22]: I don't see here the same meaning, we could discuss about that

**Review by JeremJR — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/VPNConfiguration.adoc (hunk 2 above)
Comment (4rthurLegrand) [reply modules/ROOT/pages/VPNConfiguration.adoc:39]: 5 isn't available while configuring "Phase 1 DH group numbers" AWS VPN, and 2 is available in phase 1 and 2.

**Review by JeremJR — COMMENTED** [review]
LGTM

**Review by JeremJR — APPROVED** [review]

File: modules/ROOT/pages/VPNConfiguration.adoc (hunk 3 above)
Comment (4rthurLegrand) [reply modules/ROOT/pages/VPNConfiguration.adoc]: As seen together this morning, we changed the wording.

**Pull request merged**